WeakAuras/Transmission.lua

```diff
@@ -306,7 +306,7 @@ radioButtons = {
 local radioButtonLabels = {
   L["Create a Copy"],
   -- L["Replace Aura"],
-  L["Update Aura"]
+  L["Update Selected Settings"]
```

Comment:
I'm not sure about "Settings" in this case since it's a lot of data, not really only "Settings", leave it at "Aura" for now or try "Aura's ..."